project_config.yml

```
@@ -1,3 +1,4 @@
-dataset_location: /Volumes/dbw-dplabdlz-dev-uks-01/chris_training/sample_data/twitter_sentiment.csv
+dataset_location: mlops_students.chriswroe8.twitter_sentiment
 train_pc: 0.8
-test_pc: 0.2
\ No newline at end of file
+test_pc: 0.2
+max_Iter: 10
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)